controller/loginController_test.go

```
@@ -0,0 +1,378 @@
+
+	suite.router = gin.Default()
+	apiGroup := suite.router.Group("/api")
+
+	loginCtrl := controller.NewLoginController()
+	loginCtrl.RegisterEndpoints(apiGroup)
+}
+
+// TearDownSuite runs once after all tests in the suite
+func (suite *LoginControllerTestSuite) TearDownSuite() {
+	if suite.sugar != nil {
+		suite.sugar.Sync()
```

golangci-lint (1.64.8)

Error return value of `suite.sugar.Sync` is not checked

(errcheck)